lcplants.py

```diff
@@ -1,8 +1,11 @@
 import csv
+import io
 import json
 import pprint as pp
+import sys
```

Comment:
I should have deleted this line. I was using it with `csv.field_size_limit(sys.maxsize)`, which you might still want to do if it works for you!